src/app/(mainPage)/layout.tsx

```diff
@@ -0,0 +1,12 @@
+import ClientSideLayout from '@src/components/sharedComponents/layout/ClientSideLayout';
+import React, { FC, PropsWithChildren } from 'react';
+
+const MainPageLayout: FC<PropsWithChildren> = ({ children }) => {
+  return (
+    <>
+      <ClientSideLayout>{children}</ClientSideLayout>
+    </>
```

Comment:
```suggestion
    <ClientSideLayout>{children}</ClientSideLayout>
```

Có 1 element thì dùng Fragment chi?